testing/demo/pkg/setup/tendermint.go

```diff
@@ -45,8 +46,18 @@ func CreateTendermintLightClient() error {
 // deployEurekaContracts deploys all of the IBC Eureka contracts (including the
 // SP1 ICS07 Tendermint light client contract) on the EVM roll-up.
 func deployEurekaContracts() error {
+	err := godotenv.Load()
+	if err != nil {
+		return fmt.Errorf("error loading .env file: %v", err)
+	}
+	prover := os.Getenv("SP1_PROVER")
+	fmt.Printf("SP1_PROVER: %s\n", prover)
+
 	cmd := exec.Command("forge", "script", "E2ETestDeploy.s.sol:E2ETestDeploy", "--rpc-url", "http://localhost:8545", "--private-key", "0x82bfcfadbf1712f6550d8d2c00a39f05b33ec78939d0167be2a737d691f33a6a", "--broadcast")
 	cmd.Env = append(cmd.Env, "PRIVATE_KEY=0x82bfcfadbf1712f6550d8d2c00a39f05b33ec78939d0167be2a737d691f33a6a")
+	if prover == "mock" {
+		cmd.Env = append(cmd.Env, "VERIFIER=mock")
+	}
```

Comment:
This is where we instruct the deployment script to deploy a mock verifier because these lines: https://github.com/cosmos/solidity-ibc-eureka/blob/8dad0c2502a8f5bc9cc970363eae115667f34754/scripts/E2ETestDeploy.s.sol#L61-L62

Reply:
Should we update the README as `VERIFIER` is no longer manually managed.

> 4. Create and populate the .env file in solidity-ibc-eureka
> ```
> # Modify the .env file:
> # Set VERIFIER=""
> ```